DataDefinitions/Properties/Commodities.resx

```diff
@@ -1458,7 +1458,7 @@
     <comment>Commodity KachiriginLeaches</comment>
   </data>
   <data name="VanayequiRhinoFur" xml:space="preserve">
-    <value>Vanayequi Ceratomorpha Fur</value>
+    <value>Vanayequi Rhino Fur</value>
```

Comment:
Inadvertent change? Should be Ceratomorpha.

Reply:
When I was adding 'Unknown', I saw these two entries where duped and interpreted it as a typo because the comment said' rhino'.

Of course it can be removed if it creates any conflict but it still needs to be resolved and writing a separate PR seemed silly.

Reply:
It's already one of several items de-duped by #646.

Reply:
In general it is better to keep each PR focussed on its stated purpose and not mix other stuff in. If you spot something, write a quick ticket and move on.

Reply:
This did, as expected, conflct. I'll handle it.